app.ts

```diff
@@ -19,7 +19,7 @@ app.use(cors());
 app.use(express.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 // authorization middleware
-// app.use(authUser);
+app.use(authUser);
```

Comment:
comment out this line, we will add the middleware when the frontend is ready

Reply:
Hey, it would be better to test with the middleware active... We need createdBy, DeletedBy, etc fields to be populated, Also I have set the variables in Postman so when you login the token is available to every other route. So it would allow us to know the complete flow.